application/app.py

```diff
@@ -198,12 +199,35 @@ def images_exist(self, folder):
         return False
 
 
+    def upload(self):
+        project_name = ac.CURRENT_PROJECT_PATH.split('/')[-1].strip('/')
```

Comment:
I removed .strip('/') from the newest commit since there is no difference if the path is in the form `path/to/file`